php/Terminus/Models/Collections/Instruments.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Terminus\Models\Collections;
+
+class Instruments extends TerminusCollection {
+
+  /**
+   * Instantiates the collection, sets param members as properties
+   *
+   * @param [array] $options To be set to $this->key
+   * @return [Instruments] $this
+   */
+  public function __construct($options = array()) {
+    $option_args = array_merge(
+      $options, 
+      array(
+        'fetch_url'  => 'users/' . $options['user']->get('id') . '/instruments',
+        'owner_name' => 'user',
+      )
+    );
+    parent::__construct($option_args);
```

Comment:
I don't know if this is just me or not, but I tend to like calling super at the top of a sub class's method, rather than the bottom. Right now it doesn't really matter though.